airship_tempest_plugin/services/shipyard/json/document_staging_client.py

```diff
@@ -22,6 +22,7 @@
 
 from tempest.lib.common import rest_client
 
+
```

Comment:
is the extra line needed?

Reply:
Extra line is removed.

Reply:
I don't see it as removed. Perhaps I am not seeing the latest updates made?